espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/UiautomatorParams.java

```diff
@@ -0,0 +1,166 @@
+package io.appium.espressoserver.lib.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+public class UiautomatorParams extends AppiumParams {
+    private Strategy strategy;
+    private String value;
+    private Integer index;
+    private Action action;
+
+    public Action getAction() {
+        return action;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public Strategy getStrategy() {
+        return strategy;
+    }
+
+    @Nullable
+    public Integer getIndex() {
+        return index;
+    }
+
+    public enum Strategy {
+        @SerializedName("clazz")
+        CLASS_NAME("clazz"),
```

Comment:
Please use constants instead of hardcoded strings, for example
`By.clazz.class.getSimpleName()`

Reply:
Can't get this working. clazz is a method and it doesn't have direct properties

Reply:
yep, makes sense. Please ignore